requirements.txt

```diff
@@ -15,7 +15,7 @@ commonmark==0.5.4
 docutils==0.13.1
 mock==1.0.1
 pillow==2.6.1
-Pygments==2.2.0
+Pygments==2.3.1
```

Comment:
was this intended to change? i'd include a commit message + possibly separate commit for it if so

Reply:
ugh no that wasn't meant to be included. taking it out. thanks!